spec/template.dd

```diff
@@ -637,6 +642,44 @@ $(GNAME TemplateAliasParameterDefault):
         }
         ------
         )
+
+        $(LI Compile-time values
+
+        ------
```

Comment:
You could wrap the block with this

```
$(SPEC_RUNNABLE_EXAMPLE_RUN
---
...
---
)
```

then the spec tester would verify the (static) asserts.
There's also SPEC_RUNNABLE_EXAMPLE_COMPILE